### ISSUE TITLE
Optimization of Latent Vector for SVR Refinement 

### ISSUE THREAD

**Issue by b7leung** [edoRemelli/MeshSDF#5]
I noticed that when I plot the losses for SVR refinement, the optimization of the loss is not easy. For example, for the Chair class, refinement of a few instances look like the first plot below, but many also look like the 2nd and 3rd plots. Is this a reasonable, expected behavior for SVR refinement? Or am I doing something incorrectly? I kept the hyperparameters the same (eg, learning rate is 5e-5).

![download](https://user-images.githubusercontent.com/20059131/118383935-45d28800-b5b7-11eb-9405-57dba0aad7d0.png)
![download (1)](https://user-images.githubusercontent.com/20059131/118383937-466b1e80-b5b7-11eb-93a7-4e25633da9be.png)
![download (2)](https://user-images.githubusercontent.com/20059131/118383938-466b1e80-b5b7-11eb-8fab-cafbc23bb376.png)

**Comment by bguillard** [comment]
Indeed these kinds of issues can happen. We reported average improvements in the paper, but sometimes (rarely) the predicted shape is not improved.
